src/components/elements/DateWithRelativeTooltip.tsx

```diff
@@ -54,7 +54,10 @@ const DateWithRelativeTooltip = ({
       >
         {formattedDate}
         {/* Include the tooltip text as visually hidden for accessibility */}
-        <Box sx={visuallyHidden}>, {formattedDateRelative}</Box>
+        {/* Add position: fixed to address visual bug when visuallyHidden is used inside dialog */}
```

Comment:
Seems like when `visuallyHidden` is used inside a dialog, it causes some surprising scroll behavior. I wasn't able to repro it on stackblitz using the latest mui version. I would propose cutting a separate ticket to track this down later (and see if it impacts anywhere else in our app, since we use `visuallyHidden` all over the place)